DSC/DSC_to_PIF_NETZSCH_3500.py

```diff
@@ -0,0 +1,88 @@
+import csv
```

Comment:
Let's try to follow naming conventions on module and package names: https://www.python.org/dev/peps/pep-0008/#prescriptive-naming-conventions